### DISCUSSION TITLE
Using librechat.ymlinstead of .env

### DISCUSSION THREAD

**Discussion by JayDoubleu** [danny-avila/LibreChat#3169] [General]
Is there a way to configure claude and other LLMs from .env but within librechat.yml

**Comment by prononext** [comment]
Hi @JayDoubleu yes all possible following the docs here: https://www.librechat.ai/docs/configuration/librechat_yaml

How to add config: https://www.librechat.ai/docs/configuration/librechat_yaml/ai_endpoints

Sometimes you need to check the LLMs documentation to find the correct API url.
